.github/scripts/trymerge.py

```diff
@@ -870,7 +871,7 @@ def merge_on_green(pr_num: int, repo: GitRepo, dry_run: bool = False) -> None:
             print(f"Merged failed due to: {ex}. Retrying in 60 seconds.")
             time.sleep(60)
     # Finally report timeout back
-    msg = "Merged timed out after 6 hours. Please contact the pytorch_dev_infra team."
+    msg = "Merged timed out after 400 minutes. Please contact the pytorch_dev_infra team."
```

Comment:
Nit: can you make a timeout_min default argument of `merge_on_green` and change this one into f-string?
```suggestion
    msg = f"Merged timed out after {timeout_min} minutes. Please contact the pytorch_dev_infra team."
```